pages/join/[...joinCode].js

```diff
@@ -12,8 +12,15 @@ export default function JoinWithCode() {
       method: 'PUT',
       body: JSON.stringify(formData)
     });
-    alert('Successfully registered, please extit window');
-    return await response.json();
+    const responseJson = await response.json();
+    if (responseJson == 400) {
+      alert('This email is not associated with freeCodeCamp');
+    } else if (responseJson == 409) {
```

Comment:
```suggestion
    const status = response.status;
    if (status == 400) {
      alert('This email is not associated with freeCodeCamp');
    } else if (status == 409) {
```
Since we're setting the status.

Reply:
Also did this in the new commit.